opentelemetry-user-events-trace/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "opentelemetry-user-events-trace"
 description = "OpenTelemetry-Rust exporter to user_events"
-version = "0.10.0"
```

Comment:
Oh, that was not copy-paste, but I thought of directly publish the much stable 0.10.0 version :)

Reply:
since tracing signal itself is far from stable upstream, this can't go stable anyway. I think 0.1.0 is good first release, and we can totally skip few to match the versions across etw/user-event exporters too.

Reply:
My comment was in lighter note, it was copy paste error :)

Reply:
I need ☕ before reviewing Github 🤣